precompiles/crowdloan-rewards/CrowdloanInterface.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.0;
+
+/// The interface through which solidity contracts will interact with Crowdloan Rewards
+/// We follow this same interface including four-byte function selectors, in the precompile that
+/// wraps the pallet
+interface CrowdloanRewards {
+    // First some simple accessors
+    /// Checks whether the address is a contributor
```

Comment:
would be nice to use solidity comment annotations but this could be done for all the `sol` contracts in a follow up https://jeancvllr.medium.com/solidity-tutorial-all-about-comments-bc31c729975a